.editorconfig

```diff
@@ -1,9 +1,9 @@
 root = true
 
 [*]
-indent_style = space
-indent_size = 2
+indent_style = tab
+indent_size = unset
```

Comment:
Let's keep this the same. I meant adding this entry:

```editorconfig
[*.sh]
indent_style = tab
indent_size = unset
```

Reply:
i'd just as soon bow to shfmt and let it decide. i'm a pretty "let the defaults do your work for you" kinda person

https://github.com/mvdan/sh#shfmt

Reply:
fixed in https://github.com/spenserblack/actions-wiki/pull/41/commits/02c0967686fddfa08224c74cb722c1bf28111600